doc/api/zlib.md

```diff
@@ -254,6 +254,42 @@ possible to determine whether the input ended prematurely or lacks the
 integrity checks, making it necessary to manually check that the
 decompressed result is valid.
 
+## Compressing multiple files together
+
+<!-- YAML
+added: REPLACEME
+-->
+
+The `zlib` library provides ways to compress individual objects, but not to
+aggregate multiple ones into a single file suitable for redistribution (what
+is often called archival).
+
+To this end, `node:zip` provides the `ZipArchive` class which allows to create,
+read, and modify zip archives:
```

Comment:
I assume this section needs updating? Because of references to `node:zip` etc.

Reply:
Indeed, it's a typo, the archive is now part of `node:zlib` (would it make sense to have a check in lint-md that all `node:something` identifiers must be valid?)